src/fillTank.test.js

```diff
@@ -1,10 +1,106 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  test('should be declared', () => {
+    expect(fillTank)
+      .toBeInstanceOf(Function);
+  });
+
+  test(`should refills to full tank`
+  + ` if the 'amount' is not given`, () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    fillTank(customer, 35);
+
+    expect(customer.vehicle.fuelRemains)
+      .toEqual(40);
+  });
+
+  test(`should refill to full tank`
+  + ` if the 'amount' is greater than the tank can accommodate`, () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    fillTank(customer, 35, 48);
+
+    expect(customer.vehicle.fuelRemains)
+      .toBe(40);
+  });
+
+  test('should refills only amount what the client can pay', () => {
+    const customer = {
+      money: 0,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 20,
+      },
+    };
+
+    fillTank(customer, 35, 15);
+
+    expect(customer.vehicle.fuelRemains)
+      .toBe(20);
+  });
```

Comment:
A client does not have money at all. Change the description of the test of test data